msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,610 @@
+---
+title: Add an Outlook Add-in to a Teams app
+description: Learn how to add an Outlook Add-in to a Teams app.
+ms.date: 01/11/2024
+ms.author: mosdevdocs
+author: rickki
+ms.topic: tutorial
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+
+# Add an Outlook Add-in to a Teams app
+
+## Introduction
+
+Office Add-ins are web apps that extend the functionality of Outlook. Some of the things that can be done with an Outlook Add-in:
+
+- Read and write the content of email messages and meeting invitations, as well as responses, cancellations, and appointments.
+- Read properties of the user's mailbox.
+- Respond automatically to events, such as the sending of an email.
+- Integrate with external services including CRM and project management.
+- Add custom ribbon buttons or menu items to perform specific tasks.
+
+For more information, start with [Outlook Add-ins Overview](/office/dev/add-ins/outlook/outlook-add-ins-overview).
+
+## Prerequisites
+
+To configure an Office Add-in as additional capability, you must meet the following conditions:
+
+- You have a Microsoft 365 account that includes Teams, or separate subscriptions to both Microsoft 365 and Teams, to test the application. For example, an *.onmicrosoft.com account.
```

Comment:
How about:

For example, an *onmicrosoft.com test tenant account through the [Microsoft 365 Developer Program](https://learn.microsoft.com/en-us/office/developer-program/microsoft-365-developer-program).